django/db/backends/sqlite3/features.py

```diff
@@ -147,6 +147,12 @@ def introspected_field_types(self):
             "SmallAutoField": "AutoField",
         }
 
+    @cached_property
+    def max_query_params(self):
+        # This limit can be lowered per connection at runtime with setlimit().
+        # If set, the cached property must be deleted to take effect.
+        return self.connection.connection.getlimit(sqlite3.SQLITE_LIMIT_VARIABLE_NUMBER)
```

Comment:
The next thing might be to get the value of `SQLITE_LIMIT_COMPOUND_SELECT` in the following: https://github.com/django/django/blob/0ee06c04e0256094270db3ffe8b5dafa6a8457a3/django/db/backends/sqlite3/operations.py#L49-L50